scripts/create-release-tag.js

```diff
@@ -69,8 +55,58 @@ async function main() {
     console.error(`Tag creation for ${tagName} failed`);
     throw ex;
   }
+}
+
+async function dispatchReleaseWorkflows(octokit, releaseVersion) {
+  console.log('Dispatching release manifest sync');
+  await octokit.actions.createWorkflowDispatch({
+    owner: 'backstage',
+    repo: 'backstage',
+    workflow_id: 'sync_release-manifest.yml',
+    ref: 'master',
+    inputs: {
+      version: releaseVersion,
+    },
+  });
+
+  console.log('Dispatching upgrade helper sync');
+  await octokit.actions.createWorkflowDispatch({
```

Comment:
Gonna update this to not make this 2nd dispatch here, since after we've updated it to use Backstage releases the version manifest must exist before we dispatch the upgrade helper diff. Thinking it makes the most sense to dispatch the sync update at the end of `sync_release-manifest`